### PR TITLE
Support ConvTranspose2d BatchNorm2d fusion

### PR DESCRIPTION
**Pull request by triple-mu**
Testing pass:

``` python
import torch
import torch.nn as nn
from mmcv.cnn.utils.fuse_conv_bn import _fuse_conv_bn
import random

random.seed(0)


@torch.no_grad()
def test_fusion():
    in_channel = random.randint(1, 100)
    out_channels = random.randint(1, 100)
    kernel_size = random.randint(1, 7)
    stride = random.randint(1, 2)
    padding = random.randint(0, 1)
    height = random.randint(32, 640)
    width = random.randint(32, 640)
    batch = random.randint(1, 32)
    conv = nn.Conv2d(in_channel, out_channels, kernel_size=kernel_size, stride=stride, padding=padding)
    cbn = nn.BatchNorm2d(out_channels)
    conv.eval()
    cbn.eval()

    deconv = nn.ConvTranspose2d(in_channel, out_channels, kernel_size=kernel_size, stride=stride, padding=padding)
    dbn = nn.BatchNorm2d(out_channels)
    deconv.eval()
    dbn.eval()

    x = torch.randn(batch, in_channel, height, width)

    conv_out = cbn(conv(x))
    fconv = _fuse_conv_bn(conv, cbn)
    fconv_out = fconv(x)
    torch.testing.assert_close(conv_out, fconv_out, rtol=1e-5, atol=1e-5)

    dconv_out = dbn(deconv(x))
    fdeconv = _fuse_conv_bn(deconv, dbn)
    fdconv_out = fdeconv(x)
    torch.testing.assert_close(dconv_out, fdconv_out, rtol=1e-5, atol=1e-5)


def main():
    for _ in range(100):
        test_fusion()


if __name__ == '__main__':
    main()

```

### CODE REVIEW

**Review by HAOCHENYE — COMMENTED** [review]
Hi, the corresponding unit test should also be updated to make sure the implementation is right.
